server/src/graphql/index.ts

```diff
@@ -6,7 +6,9 @@ import { commentsTypeDefs } from "../comments/Schemas";
 import { commentsResolvers } from "../comments/Resolvers";
 import { usersTypeDefs } from "../users/Schemas";
 import { usersResolvers } from "../users/Resolvers";
+import cookie from "cookie";
 
+// TODO: verify cors
```

Comment:
Validate cors configuration. Check if it can be removed and how will it work from iframe.